src/cider/nrepl/middleware/debug.clj

```diff
@@ -12,10 +12,10 @@
    [nrepl.middleware.session :as session]
    [nrepl.misc :refer [response-for]]
    [nrepl.transport :as transport]
-   [orchard.info :as info]
-   [orchard.inspect :as inspect]
-   [orchard.meta :as m]
-   [orchard.misc :as misc])
+   [mranderson049.orchard.v0v4v42.orchard.info :as info]
```

Comment:
This looks like some bug to me.

Reply:
I will change this, I wonder why I changed it